docs/modules/ROOT/pages/references/architecture/upgrade_controller.adoc

```diff
@@ -193,9 +245,17 @@ spec:
       clusterID: bc75be34-e92d-4745-bb9d-8ec39e877854
       desiredUpdate: {} <2>
       upstream: https://api.openshift.com/api/upgrades_info/v1/graph
+  overlays: <3>
+    - from: "2024-07-12T00:00:00Z"
+      overlay:
+        spec:
+          template:
+            spec:
+              channel: stable-4.15 <4>
```

Comment:
This doesn't match the structure shown above, is it `overlay.spec.template.spec` or `overlay.spec` to configure `spec` of the OpenShift ClusterVersion object?

Reply:
it's `overlay.spec`. fixed and improved the comments on the manifests.